src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -64,6 +64,11 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.debug')->addTag('twig.extension');
         }
 
+        if ($container->has('twig.loader.filesystem')) {
```

Comment:
I think this check is not useful. The service should exist and is loaded by the bundle extension.

We do not have such check [here](https://github.com/symfony/symfony/pull/20727/files#diff-65cf25daa28d5879496797054479dd41R32) for instance.